consent-ui/src/app/ais/entry-page/initiation/transactions/transactions-consent-review/transactions-consent-review.component.ts

```diff
@@ -69,4 +84,34 @@ export class TransactionsConsentReviewComponent implements OnInit {
   onBack() {
     this.location.back();
   }
+
+  private createForm() {
+    this.targetForm = this.formBuilder.group({
```

Comment:
Please rename targetForm to consentReviewForm

Reply:
renamed